src/main/java/net/dv8tion/jda/core/hooks/ListenerAdapter.java

```diff
@@ -174,6 +174,7 @@ public void onTextChannelUpdateTopic(TextChannelUpdateTopicEvent event) {}
     public void onTextChannelUpdatePosition(TextChannelUpdatePositionEvent event) {}
     public void onTextChannelUpdatePermissions(TextChannelUpdatePermissionsEvent event) {}
     public void onTextChannelUpdateNSFW(TextChannelUpdateNSFWEvent event) {}
+    public void onTextChannelUpdateParentEvent(TextChannelUpdateParentEvent event) {}
```

Comment:
This should be `onTextChannelUpdateParent` to be consistent with the rest of this class.

Reply:
whoops